### ISSUE TITLE
Accessibility test - base

### ISSUE THREAD

**Issue by mwllr** [ATLAS-Illinois/Illinois-Framework-Theme#475]
from keith
https://drupal.webtheme.illinois.edu/accessibility-testing/undergraduate-financial-support

* [ ] The heading for the left nav must be an H2, not an H3. All new sections / regions must begin with H2.
  * The "In This Section" heading could remain in the page but be hidden visually to correct this issue.
* [x] &lt;nav&gt; regions must be uniquely labelled.
* [ ] The aria-labelledby  property of the &lt;il-section-nav&gt; element does not appear to point to an ID that exists in the page. Further this property must be on the &lt;nav&gt; element for the menu, not a container element.
* [x] Remove role=link from the &lt;div&gt; containing the skip to main link. The link role is a widget role that should only be used when creating a custom link widget.
* [x] Complementary region (left nav &lt;aside&gt;) must not be inside the main region.
  * This is a common Drupal template issue to look out for--sidebars are often implemented as &lt;aside&gt; elements, which have a complementary role by default. Move the &lt;aside&gt; outside the &lt;main&gt; element or change the &lt;aside&gt; to a &lt;div&gt;.
   * Note: either way, remove role=complementary. This is redundant ARIA use.
* [x] #520
* [x] The skip to main link should jump to the &lt;main&gt; element, not a named link.
  * The named link with label "Start of Content Region" is redundant and should be removed.
  * Screen readers will announce a region name when a user jumps to it, the name and "Region Start" when entering a region (forward navigation), and the name and "Region End" when leaving the region.
* [ ] The site search form should have a role of search. This can go on the &lt;form&gt; element or its immediate container.
* [ ] The top-level links of the main navigation menu must have the aria-expanded property, and the value must toggle between expanded and collapsed as the child menus are displayed or dismissed.
  * The aria-expanded property must be on the &lt;a&gt; element, not a container (e.g. the element that receives browser focus).

Most of these issues are flagged automatically with AInspector WCAG for Firefox. It is best to run that tool and fix issues found prior to requesting a manual evaluation.

**Comment by khays9** [comment]
copied to top

* [ ] The heading for the left nav must be an H2, not an H3. All new sections / regions must begin with H2.
  * The "In This Section" heading could remain in the page but be hidden visually to correct this issue.
* [ ] &lt;nav&gt; regions must be uniquely labelled.
* [ ] The aria-labelledby  property of the &lt;il-section-nav&gt; element does not appear to point to an ID that exists in the page. Further this property must be on the &lt;nav&gt; element for the menu, not a container element.
* [ ] Remove role=link from the &lt;div&gt; containing the skip to main link. The link role is a widget role that should only be used when creating a custom link widget.
* [ ] Complementary region (left nav &lt;aside&gt;) must not be inside the main region.
  * This is a common Drupal template issue to look out for--sidebars are often implemented as &lt;aside&gt; elements, which have a complementary role by default. Move the &lt;aside&gt; outside the &lt;main&gt; element or change the &lt;aside&gt; to a &lt;div&gt;.
   * Note: either way, remove role=complementary. This is redundant ARIA use.
* [ ]  &lt;nav&gt; regions in the footer must not be empty. If a region has not content, it should not be in the page.
* [ ] The skip to main link should jump to the &lt;main&gt; element, not a named link.
  * The named link with label "Start of Content Region" is redundant and should be removed.
  * Screen readers will announce a region name when a user jumps to it, the name and "Region Start" when entering a region (forward navigation), and the name and "Region End" when leaving the region.
* [ ] The site search form should have a role of search. This can go on the &lt;form&gt; element or its immediate container.
* [ ] The top-level links of the main navigation menu must have the aria-expanded property, and the value must toggle between expanded and collapsed as the child menus are displayed or dismissed.
  * The aria-expanded property must be on the &lt;a&gt; element, not a container (e.g. the element that receives browser focus).

Most of these issues are flagged automatically with AInspector WCAG for Firefox. It is best to run that tool and fix issues found prior to requesting a manual evaluation.

**Comment by wrg20** [comment]
* [ ] The heading for the left nav must be an H2, not an H3. All new sections / regions must begin with H2.
  * The "In This Section" heading could remain in the page but be hidden visually to correct this issue.  
* [ ] The aria-labelledby  property of the &lt;il-section-nav&gt; element does not appear to point to an ID that exists in the page. Further this property must be on the &lt;nav&gt; element for the menu, not a container element.
* [ ] The site search form should have a role of search. This can go on the <form> element or its immediate container.
* [ ] The top-level links of the main navigation menu must have the aria-expanded property, and the value must toggle between expanded and collapsed as the child menus are displayed or dismissed.
* The aria-expanded property must be on the <a> element, not a container (e.g. the element that receives browser focus).



These are located in the web toolkit components and should be moved to their issue list. 
https://cdn.brand.illinois.edu/toolkit/dev/examples/components/section-nav/index.html

**Comment by wrg20** [comment]
* [ ] &lt;nav&gt; regions must be uniquely labelled.

Added aria-label to "additional header links" and "subfooter links"

**Comment by wrg20** [comment]
* [ ] Complementary region (left nav &lt;aside&gt;) must not be inside the main region.
  * This is a common Drupal template issue to look out for--sidebars are often implemented as &lt;aside&gt; elements, which have a complementary role by default. Move the &lt;aside&gt; outside the &lt;main&gt; element or change the &lt;aside&gt; to a &lt;div&gt;.
   * Note: either way, remove role=complementary. This is redundant ARIA use.

aside element was changed to a div element and the role was removed

**Comment by wrg20** [comment]
@mwllr the other open issues are for the components team...I converted my last remaining task to an issue.
